### PR TITLE
Build cap-openstack images periodically

### PR DESCRIPTION
**Pull request by flaper87**

### CODE REVIEW

**Comment by flaper87** [comment]
This addresses https://github.com/kubernetes-sigs/cluster-api-provider-openstack/issues/49

**Comment by mrhillsman** [comment]
@flaper87 currently there are periodic jobs running. once they are done we will reload the configuration to start working with the kubernetes-sigs/cluster-api-provider-openstack repo

**Comment by flaper87** [comment]
@mrhillsman openlab doesn't seem to be happy with my patch :sob:

**Comment by kiwik** [comment]
recheck

**Comment by kiwik** [comment]
recheck

File: zuul.d/projects.yaml [hunk 1]
```diff
@@ -110,6 +110,14 @@
             branches: master
         - docker-machine-0.15.0-functional-devstack-rocky:
             branches: master
+
+- project:
+    name: kubernetes-sig/cluster-api-provider-openstack
```
Comment (kiwik) [inline zuul.d/projects.yaml]: s/kubernetes-sig/kubernetes-sigs/

This wrong project name cause CI failing, and I have added `kubernetes-sigs/cluster-api-provider-openstack` into zuul main.yaml config file, update your patch and input `recheck` in PR comments, I think everything should be ok.

Comment (flaper87) [reply zuul.d/projects.yaml]: ooops! Thanks a lot!

**Comment by flaper87** [comment]
@mrhillsman I've requested to install @theopenlab-ci in the `cluster-api-provider-openstack` repo. I'll let you know when that's done. Can this PR go in anyway?

**Comment by kiwik** [comment]
Hi @flaper87 , docker image building job have been fixed in #338 , I saw the latest job is successful in [OpenLab log] and image have been pulled into [docker hub]

[docker hub]: https://hub.docker.com/r/k8scloudprovider/openstack-cluster-api-controller/tags/
[OpenLab log]: http://status.openlabtesting.org/t/openlab/builds.html?job_name=cluster-api-provider-openstack-image-build

**Pull request merged**